libnymea-zigbee/zigbeenodeendpoint.cpp

```diff
@@ -230,6 +230,13 @@ ZigbeeCluster *ZigbeeNodeEndpoint::createCluster(ZigbeeClusterLibrary::ClusterId
     case ZigbeeClusterLibrary::ClusterIdThermostat:
         return new ZigbeeClusterThermostat(m_network, m_node, this, direction, this);
 
+        // Manufacturer specific
+    case ZigbeeClusterLibrary::ClusterIdManufacturerSpecificPhilips:
+        // FIXME: We should check the manufacturer when using manufacturer specific cluster ids, however at this point the manufacturer is not set yet.
```

Comment:
@t-mon I've tried to check for manufacturer here, but it's not possible.
The manufactuer name is set here https://github.com/nymea/nymea-zigbee/blob/master/libnymea-zigbee/zigbeenetworkdatabase.cpp#L154

But this cluster is already created a few lines up at https://github.com/nymea/nymea-zigbee/blob/master/libnymea-zigbee/zigbeenetworkdatabase.cpp#L121

I suppose this could be fixed by changing the code to always start with creating the basic cluster, process that, and then do the loops over the remaining input and output clusters. wdyt?